hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -145,6 +193,14 @@
 	  name: "bat_disable",
 	  desc: "0: don't invert; 1: invert",
 	}
+        { bits: "10",
+	  name: "lid_open",
+	  desc: "0: don’t invert; 1: invert",
```

Comment:
You're using a non-ASCII apostrophe here.

Reply:
I don't know how to tell non-ASCII apostrophe in my editor(vi). Just copy-and-paste from the previous text.